app/src/styles/blocks/cabinet.scss

```diff
@@ -0,0 +1,368 @@
+.cabinet_page {
+  padding: 0;
+  position: absolute;
+
+  .error__message {
+    margin-top: 3px;
+    width: 100%;
+    max-width: 520px;
+    padding: 30px;
+    background-color: rgba(247, 0, 0, 20%);
+    border: none;
+    border-radius: 10px;
+    justify-self: center;
+    @include button;
+
+    @include Phone {
+      grid-column: 2/12;
+    }
+
+  }
+
+  .success__message {
+    height: 60px;
+    width: 300px;
+    position: relative;
+    top: 0;
+    right: 0;
+    padding: 30px;
+    background-color: #88B13E;
```

Comment:
pls cache all the color values into a variable.
Add variables into _vars.scss and use varaibles here